src/presentation/controllers/ingest-controller.ts

```diff
@@ -0,0 +1,24 @@
+import { BaseController, PostRequest, RestController } from './base-controller'
+import { HttpErrorHandler } from '../interfaces/http-error-handler'
+import { Exception } from '../../model/exceptions/exception'
+import { Request, Response } from 'express'
+import { IngestService } from '../../business-logic/services/interfaces/ingest-service'
+
+@RestController('/ingest')
+export class IngestController extends BaseController {
+
+  constructor(private readonly ingestService: IngestService, private readonly httpErrorHandler: HttpErrorHandler) {
+    super()
+  }
+
+  @PostRequest('/reloadData/:rundownName')
```

Comment:
This is the endpoint for clients to call SofieServer. In SofieServer we use Ids.
This should be using `rundownId` and not `rundownName`.

Reply:
And to follow REST we need to prepend the plural form of the entity of which we are parsing an id.
So it should be `reloadData/rundowns/{rundownId}`

Reply:
It has been changed to follow this!